### ISSUE TITLE
Accessibility issue: Scrollable region must have keyboard access on /notebook-ci-testing/notebooks/DrizzlePac/drizzle_wfpc2/drizzle_wfpc2.html

### ISSUE THREAD

**Issue by dulude** [spacetelescope/notebook-ci-testing#463]
## What
An accessibility scan flagged 18 elements on https://spacetelescope.github.io/notebook-ci-testing/notebooks/DrizzlePac/drizzle_wfpc2/drizzle_wfpc2.html because scrollable region must have keyboard access. Learn more about why this was flagged by visiting https://dequeuniversity.com/rules/axe/4.11/scrollable-region-focusable?application=playwright.

The following elements need attention:

- `<pre id="codecell0">` (selector: `#codecell0`)
- `<pre id="codecell1">` (selector: `#codecell1`)
- `<pre id="codecell2">` (selector: `#codecell2`)
- `<pre id="codecell3"><span></span>Drizzled Image                             Dimensions    Scale     Input      Updated
                                                        ("/pix)    File       WCS?
</pre>` (selector: `#codecell3`)
- `<pre id="codecell9">` (selector: `#codecell9`)
- `<pre id="codecell11">` (selector: `#codecell11`)
- `<pre id="codecell14">` (selector: `#codecell14`)
- `<pre id="codecell16">` (selector: `#codecell16`)
- `<pre id="codecell18">` (selector: `#codecell18`)
- `<pre id="codecell23">` (selector: `#codecell23`)
- `<table id="table139856690703760" class="table-striped table-bordered table-condensed">` (selector: `#table139856690703760`)
- `<pre id="codecell24">` (selector: `#codecell24`)
- `<pre id="codecell28">` (selector: `#codecell28`)
- `<pre id="codecell29">` (selector: `#codecell29`)
- `<pre id="codecell32">` (selector: `#codecell32`)
- `<pre id="codecell39">` (selector: `#codecell39`)
- `<pre id="codecell44">` (selector: `#codecell44`)
- `<pre id="codecell45">` (selector: `#codecell45`)

To fix this, ensure elements that have scrollable content are accessible by keyboard in safari.

Specifically:

Fix any of the following:
-   Element should have focusable content
-   Element should be focusable

## Acceptance Criteria
- [ ] The specific violation reported in this issue is no longer reproducible.
- [ ] The fix MUST meet WCAG 2.2 guidelines OR the accessibility standards specified by the repository or organization.
- [ ] A test SHOULD be added to ensure this specific violation does not regress.
- [ ] This PR MUST NOT introduce any new accessibility issues or regressions.
